sonar-java-plugin/src/main/resources/static/documentation.md

```diff
@@ -63,6 +63,15 @@ Example: `sonar.java.source=1.6`
 
 If the property is provided, the analysis will take the source version into account, and execute related rules accordingly. At run time, each of these rules will be executed – or not – depending of the Java version used by sources within the project. For instance, on a correctly configured project built with Java 6, rules targeting Java 7 and Java 8 will never raise issues, even though they are enabled in the associated rule profile.
 
+## Batch mode settings
+
+By default, files are parsed in batches. The size of the batch is dynamically computed based on the maximum memory available.
+It is possible to manually set this value by using the property `sonar.java.experimental.batchModeSizeInKB`.
+Note that the perfect value depends on the project and the ecosystem setup, bigger batch size will not necessarily increase the performance and can even slow things down if the memory is a limiting factor.
+If needed, it is possible to run the parsing file by file by setting `sonar.java.fileByFile=true`.
+
+More details can be found in the [wiki of the analyzer](https://github.com/SonarSource/sonar-java/wiki/Batch-mode).
```

Comment:
Personally, I won't use the wording "wiki of the analyzer".
Instead just use: 

`More details can be found [here](https://github.com/SonarSource/sonar-java/wiki/Batch-mode)`